apps/framework-cli/src/cli/routines/metrics_console/run_console/ui.rs

```diff
@@ -104,64 +107,303 @@ pub fn render(app: &mut App, frame: &mut Frame) {
     }
 }
 
-fn render_table(app: &mut App, frame: &mut Frame, layout: Rect) {
+fn render_endpoint_table(app: &mut App, frame: &mut Frame, layout: Rect) {
     let mut rows: Vec<Row> = vec![];
 
-    for x in &app.summary {
-        rows.push(
-            if *app.path_requests_per_sec.get(&x.path).unwrap_or(&0.0) > 0.0 {
-                Row::new(vec![
-                    format!("{}", x.path.to_string()),
-                    format!(
-                        "{}",
-                        ((((x.latency_sum / x.request_count) * 1000.0) * 1000.0).round() / 1000.0)
-                            .to_string()
-                    ),
-                    format!(
-                        "{}",
-                        (((x.request_count * 1000.0).round()) / 1000.0).to_string()
-                    ),
-                ])
-                .bold()
-                .magenta()
-            } else {
-                Row::new(vec![
-                    format!("{}", x.path.to_string()),
-                    format!(
-                        "{}",
-                        ((((x.latency_sum / x.request_count) * 1000.0) * 1000.0).round() / 1000.0)
-                            .to_string()
-                    ),
-                    format!(
-                        "{}",
-                        (((x.request_count * 1000.0).round()) / 1000.0).to_string()
-                    ),
-                ])
-                .green()
-                .not_bold()
-            },
-        )
+    match &app.table_state {
+        TableState::Endpoint => {
```

Comment:
This can probably be split into 2 functions right? one that generated the endpoint table and the other one the kafka table?